### ISSUE TITLE
accessibility: long-pressing keys doesn't work

### ISSUE THREAD

**Issue by techexplorers123** [futo-org/android-keyboard#453]
### What happened?

When using screen readers such as Talk Back, the long-press functionality on the keyboard does not work as expected. Instead of executing the long-press action, the keyboard only responds as if the key was tapped once you lift your finger.
This makes it impossible to enable the caps lock, hold to delete large texts quickly, or more functions.
Steps to reproduce:
1.  Tern on talk back or any other screen reader.
2. Try placing your finger on a key and hold it there.
Result:
The keyboard does nothing til you lift your finger, At which point the key you tried to long-press will get activated as if you just tapped it.
Expected result:
After some time, the keyboard should execute the long-press action assigned to the key like long-pressing shift terns on caps lock.
device: one plus node CE 5G, android 13.

### Keyboard Version

0.1.23.2

**Comment by abb128** [comment]
This will be fixed in the next release
